.evergreen/javaConfig.bash

```diff
@@ -0,0 +1,16 @@
+# Java configurations for evergreen
+# Defaults to Java 11
+
+export JDK8="/opt/java/jdk8"
+export JDK11="/opt/java/jdk11"
+export JDK17="/opt/java/jdk17"
+
+if [ -d "$JDK11" ]; then
```

Comment:
What's the intention here?  It seems that since JDK11 is defined to a constance on line 5, any conditional based on its value is going to always evaluate to the same thing.

Reply:
Checks that the directory actually exists. If not don't set JAVA_HOME.